checks.d/splunk_topology.py

```diff
@@ -139,6 +148,8 @@ def _dispatch_saved_search(self, instance_config, saved_search):
         # json output_mode is mandatory for response parsing
         parameters["output_mode"] = "json"
 
+        self.log.info("Dispatching saved search: %s." % saved_search.name)
```

Comment:
set both messages to debug level, I would too like to have the collector log be clean.